test.py

```diff
@@ -485,7 +485,7 @@ def __init__(self, test_no: int, shortname: str, suite) -> None:
         self.mode = suite.mode
         self.suite = suite
         # Unique file name, which is also readable by human, as filename prefix
-        self.uname = "{}.{}".format(self.shortname, self.id)
+        self.uname = "{}/{}.{}".format(self.suite.name, self.shortname, self.id)
```

Comment:
please use a dot for adding the suite name, not a slash. We're talking about .log files as well, not just .xml files.

Reply:
With that, I believe your fix will be a one liner, no other changes will be necessary.

Reply:
ok